packages/gatsby-plugin-image/README.md

```diff
@@ -1,356 +1,207 @@
 # gatsby-plugin-image (beta)
 
-This plugin is a replacement for gatsby-image. It adds [static images](#staticimage), and a [new higher-performance gatsby-image component](#gatsbyimage). It also adds [a new GraphQL resolver](#graphql-resolver) to gatsby-transformer-sharp.
+_The new Gatsby Image plugin is currently in beta, but you can try it out now_
+
+Adding responsive images to your site while maintaining high performance scores can be difficult to do manually. The Gatsby Image plugin handles the hard parts of producing images in multiple sizes and formats for you!
+
+For full documentation on all configuration options, see [the Gatsby Image Plugin reference guide](https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-plugin-image/)
 
 ## Contents
 
-- [StaticImage](#staticimage) - the new static image component
-- [GatsbyImage](#gatsbyimage) - a high-performance gatsby-image component
-- [gatsbyImageData](#graphql-resolver) - a simpler GraphQL API
+- [Installation](#installation)
+- [Using the Gatsby Image components](#using-the-gatsby-image-components)
+  - [Static images](#static-images)
+  - [Dynamic images](#dynamic-images)
+- [Migrating to gatsby-plugin-image](#migrating)
 
-## Usage
+## Installation
 
-1. Install `gatsby-plugin-image` and `gatsby-plugin-sharp`:
+1. Install `gatsby-plugin-image` and `gatsby-plugin-sharp`. Additionally install `gatsby-source-filesystem` if you are using static images, and `gatsby-transformer-sharp` if you are using dynamic images.
 
 ```shell
-npm install gatsby-plugin-image gatsby-plugin-sharp
+npm install gatsby-plugin-image gatsby-plugin-sharp gatsby-source-filesystem gatsby-transformer-sharp
 ```
 
-If you're using the new `GatsbyImage` in addition to `StaticImage`, you'll also want to install `gatsby-transformer-sharp`.
-
-2. Upgrade `gatsby` to at least `2.24.78`.
+1. Upgrade `gatsby` to at least `2.24.78`.
 
-3. Add the plugins to your `gatsby-config.js`:
+2. Add the plugins to your `gatsby-config.js`:
 
 ```javascript
 module.exports = {
   plugins: [
     `gatsby-plugin-image`,
     `gatsby-plugin-sharp`,
-    // `gatsby-transformer-sharp`
+    `gatsby-source-filesystem`, // Needed for static images
```

Comment:
Defining `gatsby-source-filesystem` like this will thrown an error:

![image](https://user-images.githubusercontent.com/16143594/105854158-d3797000-5fe6-11eb-864c-576453a4621d.png)

Reply:
Missed that. Also, they don't need to configure that one. Only need to install it.